### ISSUE TITLE
Open popups in the same sandbox by default ?

### ISSUE THREAD

**Issue by temsa** [bakulf/priv8#15]
Currently links which trigger popups can be hard to use, you don't see any warning that the popup has been blocked, and you can only use "open in the same sandbox" for popups triggered by real links, not by JS

**Comment by yukulele** [comment]
:+1:

**Comment by dogancelik** [comment]
:+1:

**Comment by MuhPirat** [comment]
:+1:

**Comment by Alfred456654** [comment]
I don't know how to make a "thumbs up" icon, but thumbs up :)

**Comment by Alfred456654** [comment]
:+1:

**Comment by MuhPirat** [comment]
@Alfred456654 :D I see what you did :D

**Comment by Alfred456654** [comment]
Yeah! I learned something and covered my tracks! \o/

**Comment by MuhPirat** [comment]
:D :+1:

**Comment by Schmaiffe** [comment]
Yes, agree! This is the issue that brought me here. On tumblr there's a ton of javascript popups that don't open in a new window, like being able to send asks from your dash. They keep getting blocked and it makes the site really difficult to use.
